src/components/Nav/index.js

```diff
@@ -222,6 +222,11 @@ const Nav = ({ handleThemeChange, isDarkTheme, path }) => {
           to: "/whitepaper/",
           shouldDisplay: contentVersion > 1.1,
         },
+        {
```

Comment:
I wonder - should this page be under "Learn" in the nav? Perhaps it should be under "Use Ethereum" 🤔

Reply:
Open to opinions on this!